src/main/java/uk/gov/companieshouse/service/AppealService.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.companieshouse.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class AppealService {
```

Comment:
I did not even expect you to have it here. I'd remove it at this point because we do not know whether we need that service, what it will be doing and what it should be called.

Reply:
I think this is just habit on my side :P You're right, this may not be needed, so no point in making assumptions